### ISSUE TITLE
How about supporting interval multiplying operation?

### ISSUE THREAD

**Issue by tristone13th** [AlexandreDecan/portion#63]
Such as `P.open(0, 3) * P.open(0, 3) == P.open(0, 9)`.

**Comment by AlexandreDecan** [comment]
Hi, 

What is the use case? It seems this makes sense for numbers only.

**Comment by tristone13th** [comment]
> Hi,
> 
> What is the use case? It seems this makes sense for numbers only.

Yes, I'm extending pandas and when two columns of a DataFrame multiply with each other, I want their ranges also to be scaled, so I think it's necessary to implement this feature.

**Comment by AlexandreDecan** [comment]
Ok, I get it but what does it mean, from a mathematical point of view, to multiply two intervals? Is there any well-defined definition of this?
